rest/service_management.go

```diff
@@ -37,6 +37,7 @@ func (s *ManagementService) App() *gimlet.APIApp {
 	app.AddRoute("/jobs/mark_complete/{name}").Version(1).Post().Handler(s.MarkComplete)
 	app.AddRoute("/jobs/mark_complete_by_type/{type}/{filter}").Version(1).Post().Handler(s.MarkCompleteByType)
 	app.AddRoute("/jobs/mark_many_complete/{filter}").Version(1).Post().Handler(s.MarkManyComplete)
+	app.AddRoute("/jobs/mark_complete_by_prefix/{prefix}/{filter}").Version(1).Post().Handler(s.MarkCompleteByPrefix)
```

Comment:
Is there a case where we wouldn't want to pass `{filter}`? Or alternatively, what's the behavior if `{filter} == ""`. Should it be all?

Reply:
yeah then the user should pass in `All`